system/web/context/Response.cfc

```diff
@@ -149,6 +215,63 @@ component accessors="true" {
 		variables.messages.addAll( arguments.message );
 		return this;
 	}
+	
+	/**
+	 * Sets the status code with a statusText for the API response
+	 * @statusCode The status code to be set
+	 * @statusText The status text to be set
+	 *
+	 * @return Returns the Response object for chaining
+	 */
+	function setStatusCode( required statusCode, statusText ) {
```

Comment:
Since this overrides the setter for the `statusCode` why not calling just `setStatus()`

Reply:
You can then shorten the arguments to :`code` and `text`